package.json

```diff
@@ -37,21 +37,23 @@
     "bnc-notify": "^1.9.1",
     "bnc-onboard": "1.36.0",
     "erc-20-abi": "^1.0.0",
+    "eslint": "^8.19.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-config-react-app": "^6.0.0",
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-react": "7.30.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "ethereum-block-by-date": "^1.4.2",
     "ethers": "^5.1.0",
+    "fs": "^0.0.1-security",
```

Comment:
**question** why is this needed?

Reply:
It's not necessarily needed, but not bad to have: https://www.npmjs.com/package/fs
> This package name is not currently in use, but was formerly occupied by another package. To avoid malicious use, npm is hanging on to the package name, but loosely, and we'll probably give it to you if you want it.

Reply:
FWIW - I think I accidentally added it, or something else added it

Reply:
hmmm. can we remove it! `fs` is already in node stdlib pretty sure